CHANGES/7191.misc

```diff
@@ -0,0 +1 @@
+Make tests pass after 2039
```

Comment:
It took me a while to understand that this is not a PR/issue number, but a year. A changelog fragment should be specifica about that, use the past tense, contain complete sentences (with a trailing period) and be more accurate in what it described — generic phrases are pointless in the changelog.

Reply:
I updated the file. Feel free to rewrite it as you see appropriate.
I'm just a one-time contributor.

Reply:
Personally, I'd just have skipped the changelog, I doubt any other users are going to care about this when checking what changed in a release.